Formula/librsvg.rb

```diff
@@ -18,6 +18,7 @@ class Librsvg < Formula
   depends_on "glib"
   depends_on "libcroco"
   depends_on "pango"
+  depends_on "vala"
```

Comment:
Shouldn't it be a build dependency only?

```suggestion
  depends_on "vala" => :build
```

Reply:
I'll look in to that, this PR is based on a very cursory glance at how librsvg is handled in other package managers.